service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionControllerTest.java

```diff
@@ -73,11 +80,28 @@ void shouldReturnUnsuccessfulResponseWithNoData() throws Exception {
     @Test
     void shouldReturnUnsuccessfulResponseWithMalformedData() throws Exception {
         mockMvc
-            .perform(post("/callback/case-submission")
+            .perform(post("/callback/case-submission/submitted")
                 .header("authorization", AUTH_TOKEN)
                 .header("user-id", USER_ID)
                 .contentType(MediaType.APPLICATION_JSON)
                 .content("Mock"))
             .andExpect(status().is4xxClientError());
     }
+
+    @Test
+    void shouldAddConsentLabelToCaseDetails() throws Exception {
```

Comment:
Can you reimplement this so that it follows pattern in `uk.gov.hmcts.reform.fpl.controllers.CaseInitiationControllerTest#shouldAddCaseLocalAuthorityToCaseData` test?

That one is better because it not only checks whether value is appended but also that other properties in case data are intact. We do not wanna start overriding case data without knowing it.